README.adoc

```diff
@@ -12,7 +12,7 @@ Our primary goals are:
 
 * Provide a radically faster and widely accessible getting started experience for all Spring development.
 * Be opinionated out of the box, but get out of the way quickly as requirements start to diverge from the defaults.
-* Provide a range of non-functional features that are common to large classes of projects (e.g. embedded servers, security, metrics, health checks, externalized configuration).
+* Provide a range of non-functional features common to large classes of projects (e.g., embedded servers, security, metrics, health checks, externalized configuration).
```

Comment:
"e.g." should be "for example." (We avoid Latin, especially the abbreviations. Not your fault, I know. If you don't fix it, I will.)